yt_dlp/extractor/common.py

```diff
@@ -789,9 +789,10 @@ def _webpage_read_content(self, urlh, url_or_request, video_id, note=None, errno
             self._downloader.to_screen(dump)
         if self.get_param('write_pages', False):
             basen = '%s_%s' % (video_id, urlh.geturl())
-            if len(basen) > 240:
+            trim_length = self._downloader.params.get('trim_file_name', 240)
+            if len(basen) > trim_length:
                 h = '___' + hashlib.md5(basen.encode('utf-8')).hexdigest()
-                basen = basen[:240 - len(h)] + h
+                basen = basen[:trim_length - 35] + h
```

Comment:
md5 digest_size is 16, therefore len(h) is 32 + 3

Reply:
leave it as `len(h)`